setup-ceph.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+cd /ceph
+git clone https://github.com/openattic/ceph .
+./install-deps.sh
+./do_cmake.sh $@
+cd /ceph/build
+make -j$(nproc)
```

Comment:
As these build steps will have to be performed manually every now and then, I wonder if it's feasible to put them into a script. I'm fine with calling the `./install-deps.sh` script, as this is part of the "container preparation". But IMHO anything that makes changes to the bind-mounted git repo (e.g. running a build) should not be performed by the container itself.

Reply:
Don't know if I understand that correctly. `setup-ceph.sh` is a script which will be available inside the container as `setup-ceph`. It enables you to call it and does those steps for you. If you don't like to call it, you'll be free to process these steps manually.

But most of these commands have to be called inside the Docker container for several reasons. All dependencies are installed inside the container, thus you've to call `install-deps.sh` inside the container. `do_cmake.sh` remembers the path it has been called in and prevents Ceph from being build if you call `make` and Ceph is not in the path it has been configured in. 
Of course you might just want to run Ceph in the Container and build it outside but I don't know if that's even possible. If it is, I wouldn't like it either, because I like to have the dependencies inside the container (not polluting my work environment).

Did I maybe miss something?